### PR TITLE
[PM-31] Connect Stats Fragment and Graph Fragment to db

### PR DESCRIPTION
**Pull request by zahin-mohammad**
- Removes dummy data from Stats Fragment and Graph Fragment
- Introduces the MVP score equation for exercises

I opted to use the existing `ExerciseController` since we are currently only displaying exercises. In the future, once the Stats Fragment needs to display more complex views we will create a specific controller for it.

### CODE REVIEW

File: app/src/main/res/navigation/navigation.xml [hunk 1]
```diff
@@ -53,7 +53,14 @@
...
             android:name="io.mochahub.powermeter.graph.GraphFragment"
             android:label="graph_fragment"
-            tools:layout="@layout/graph_fragment" />
+            tools:layout="@layout/graph_fragment" >
+        <argument
+                android:name="exerciseID"
+                app:argType="string" />
+        <argument
+                android:name="personalRecord"
+                app:argType="string" />
```
Comment (zahin-mohammad) [inline app/src/main/res/navigation/navigation.xml]: Can't have a `Double` type in `navargs`.

File: app/src/main/java/io/mochahub/powermeter/data/workout/WorkoutDao.kt [hunk 2]
```diff
@@ -14,6 +14,10 @@ interface WorkoutDao {
     @Query("SELECT * FROM workouts where workoutSessionUUID= :workoutSessionUUID")
     fun getWorkoutsByWorkoutSession(workoutSessionUUID: String): Flow<List<WorkoutWithRelation>>
 
+    @Transaction
+    @Query("SELECT * FROM workouts where exerciseUUID= :exerciseUUID")
+    fun getAllByExericse(exerciseUUID: String): Flow<List<WorkoutWithRelation>>
```
Comment (brian-norman) [inline app/src/main/java/io/mochahub/powermeter/data/workout/WorkoutDao.kt]: Typo:
```suggestion
    fun getAllByExercise(exerciseUUID: String): Flow<List<WorkoutWithRelation>>
```

Comment (brian-norman) [reply app/src/main/java/io/mochahub/powermeter/data/workout/WorkoutDao.kt]: Also I think you want workouts to return in order by date

File: app/src/main/java/io/mochahub/powermeter/graph/GraphFragment.kt [hunk 3]
```diff
@@ -51,12 +56,28 @@ class GraphFragment : Fragment() {
         }
 
         viewModel.data.observe(viewLifecycleOwner, Observer {
-            graphAdapter.setData(it ?: emptyList())
+            if (it.size < 5) {
+                showEmptyState()
```
Comment (brian-norman) [inline app/src/main/java/io/mochahub/powermeter/graph/GraphFragment.kt:60]: Why 5?

File: app/src/main/java/io/mochahub/powermeter/graph/GraphFragment.kt [hunk 4]
```diff
@@ -51,12 +56,28 @@ class GraphFragment : Fragment() {
...
+        graph.visibility = View.INVISIBLE
+        scrub_value.visibility = View.INVISIBLE
+        emptyStateText.visibility = View.VISIBLE
+    }
+
+    private fun showGraph() {
+        graph.visibility = View.VISIBLE
+        scrub_value.visibility = View.VISIBLE
+        emptyStateText.visibility = View.INVISIBLE
+    }
```
Comment (brian-norman) [inline app/src/main/java/io/mochahub/powermeter/graph/GraphFragment.kt:82]: Epic 💯

File: app/src/main/res/navigation/navigation.xml (hunk 1 above)
Comment (brian-norman) [reply app/src/main/res/navigation/navigation.xml]: We used Float in `ExerciseFragment` to `ExerciseDialog` args, would be nice if one or the other was standardized across the app

**Review by brian-norman — CHANGES_REQUESTED** [review]

File: app/src/main/java/io/mochahub/powermeter/graph/GraphFragment.kt (hunk 3 above)
Comment (zahin-mohammad) [reply app/src/main/java/io/mochahub/powermeter/graph/GraphFragment.kt:60]: It was a random number, but from testing-checking, >5 data points make a decent starting point for the graph.

File: app/src/main/res/navigation/navigation.xml (hunk 1 above)
Comment (zahin-mohammad) [reply app/src/main/res/navigation/navigation.xml]: Changed here https://github.com/power-meter/android-client/pull/72/commits/3a834c4baf511bf0bcf4ee0df05fad312be6af38

**Review by brian-norman — APPROVED** [review]
🎉

**Pull request merged**